clients/python/src/model_registry/_experiments.py

```diff
@@ -97,8 +97,10 @@ def log_param(self, key: str, value: Any, *, description: str | None = None):
         elif isinstance(value, (int, float)):
             # TODO: ensure for numpy and other numeric types
             param_type = ParameterType.NUMBER
-        else:
+        elif isinstance(value, dict):
             param_type = ParameterType.OBJECT
+        else:
+            param_type = ParameterType.STRING
```

Comment:
I think it's already the "default" per line 94, maybe it's best to add a comment here ~like:

```suggestion
        else:
            param_type = ParameterType.STRING # consistent with param_type default init value above
```

thanks for making this consistent!